symplyphysics/laws/hydro/froude_number.py

```diff
@@ -0,0 +1,39 @@
+from sympy import Eq, solve, sqrt, S
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless, convert_to)
+
+
+# Description
+# Froude number characterizes the ratio between the force of inertia and
+# the external force, in the field of which the motion occurs, acting on
+# an elementary volume of liquid or gas.
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length (the characteristic length is the dimension
+# that defines the length scale of a physical system),
```

Comment:
Sorry but it is very hard to understand what it is. I see that you took it from Wikipedia, but could you explain it in layman terms? You can also add a waterline of a ship, as an example of characteristic length.